sct.py

```diff
@@ -458,6 +458,24 @@ def search_builder(test_id):
     click.echo(tbl.get_string(title='Found builders for Test-id: {}'.format(test_id)))
 
 
+@investigate.command('show-events', help='Return content of file events_log/events for running job by test-id')
+@click.argument('test-id')
+@click.option("--follow", type=int, required=False, help="Follow job events log file starting from N line from file end")
+@click.option("--save-to", type=str, required=False, help="Download events.log file and save to provided dir")
+def show_events(test_id: str, follow: bool = False, save_to: str = None):
+    logging.getLogger("paramiko").setLevel(logging.CRITICAL)
+    builders = get_builder_by_test_id(test_id)
+    for builder in builders:
+        remoter = builder["builder"]["remoter"]
+        if follow:
+            remoter.run(f"tail -f -n {follow} {builder['path']}/events_log/events.log")
```

Comment:
remove `-f`

Reply:
It is needed here to follow file and get any changes

Reply:
@bentsi i updated the function. Now it support follow file like tail -f, could return last n lines like tail -n 50, and could combine modes tail -f -n 50 will return last 50 lines and continue get changes